src/DynamoCore/Graph/Nodes/ZeroTouch/DSFunction.cs

```diff
@@ -49,6 +51,28 @@ public DSFunction(FunctionDescriptor functionDescription)
             : base(new ZeroTouchNodeController<FunctionDescriptor>(functionDescription)) 
         {
         }
+
+        [NodeMigration(version: "2.0.0.0")]
+        public static NodeMigrationData MigrateGetImportedObjects(NodeMigrationData data)
+        {
+            var migrationData = new NodeMigrationData(data.Document);
+            var node = data.MigratedNodes.ElementAt(0);
+
+            if (node.GetAttribute("function").Equals("Dynamo.Translation.FileLoader.GetImportedObjects"))
```

Comment:
Since `FileLoader.GetImportedObjects` doesn't belong to DynamoCore we can't really implement a migration behavior here. I was wondering what if we use migration.xml and migrate this node to [Identity node](https://github.com/DynamoDS/Dynamo/blob/ff17c04a4b58cf72b5e3bfe91a4cca6250054bec/src/Libraries/CoreNodes/Object.cs#L30)?